Makefile.toml

```diff
@@ -342,19 +342,56 @@ run_task = "run-twoliter"
 
 # A top level target for devs to ensure review and patch readiness
 [tasks.check]
+dependencies = ["check-shell"]
 run_task = "run-twoliter"
 
 [tasks.check-fmt]
 run_task = "run-twoliter"
 
 [tasks.check-lints]
+dependencies = ["check-shell"]
 run_task = "run-twoliter"
 
 [tasks.check-clippy]
 run_task = "run-twoliter"
 
 [tasks.check-shell]
-run_task = "run-twoliter"
+script = [
+'''
+rc=0
+
+# For bash first-party shell code
+if ! docker run --rm \
+  --network=none \
+  --user "$(id -u):$(id -g)" \
+  --security-opt="label:disable" \
+  -v "${BUILDSYS_TOOLS_DIR}":/tmp/tools \
+  "${BUILDSYS_SDK_IMAGE}" \
+  bash -c \
```

Comment:
This doesn't feel like the right interface given that `twoliter` is going to need to manage the SDK install and execution. `twoliter` also needs to do its own shell-checking for `rpm2img` etc, though that doesn't need to be delivered as a Makefile.toml task.

What do you think about exposing a `twoliter lint <path>` command that will run all the lints for the target path? Potentially to include Rust, Go, shell, rpm spec lints.

In the interim this could just be done in CI. We might want to consider a `Makefile` or some other build interface for the non-twoliter tasks, since it is very confusing to blend them together in the same `Makefile.toml`.

Reply:
Should we consider adding that to the Twoliter backlog and putting this back here for now?

Reply:
I'm OK with the Twoliter backlog part, but I'd rather this didn't come back here.

Losing `shellcheck` for a couple of developer-oriented scripts feels like the lesser evil compared to prolonging the existence of this `Makefile.toml` and adding new non-twoliter functionality.